src/algebra/group/defs.lean

```diff
@@ -103,6 +103,30 @@ def left_mul : G → G → G := λ g : G, λ x : G, g * x
 @[to_additive "`right_add g` denotes right addition by `g`"]
 def right_mul : G → G → G := λ g : G, λ x : G, x * g
 
+/-- A mixin for left cancellative multiplication. -/
+@[protect_proj] class is_left_cancel_mul (G : Type u) [has_mul G] :=
```

Comment:
These classes should be `Prop`-valued, `def`s below should be `lemma`s or `instance`s.

Reply:
They cannot be instances (the linter would complain), but they're now lemmas.